conn.go

```diff
@@ -195,6 +196,7 @@ func (conn *Conn) Close() error {
 		close(conn.eavesdropped)
 	}
 	conn.eavesdroppedLck.Unlock()
+	conn.outWait.Wait()
```

Comment:
I realize that, but outWait protects the outWorker, which then reads from conn.out, and it will be running until the Wait() call.